src/app.py

```diff
@@ -1,59 +1,79 @@
 import logging
 import uuid
+import uvicorn
+
+from fastapi import Body, FastAPI, Response
+
 
-from fastapi import Body, FastAPI
 
 import database
 
-logging.basicConfig(level=logging.INFO)
+from constants import OK
+from paths import (
+    HEALTHCHECK_PATH,
+    DATASET_PATH,
+    DATASETS_PATH,
+    UNIT_DATA_PATH,
+    DATASET_SCHEMA_PATH,
+    DATASET_SCHEMAS_PATH
+)
+
+
+level = logging.INFO
+
+logging.basicConfig(level=level)
 
 
 app = FastAPI()
 
 
-@app.post("/dataset")
-async def dataset(payload: dict = Body(...)):
-    """Put a dataset file into the database and return the dataset id."""
-    dataset_id = str(uuid.uuid4())
-    for sup_data in payload["data"]:
-        database.set_data(dataset_id, sup_data)
-    database.set_dataset(dataset_id, payload)
-    return {"dataset_id": dataset_id}
+@app.get(HEALTHCHECK_PATH)
+async def get_healthcheck(response: Response):
+    print("setting headers!")
 
+    response.headers["content-type"] = "text/plain"
 
-@app.get("/unit_data")
-async def unit_data(dataset_id: str, unit_id: str):
-    """Retrieve supplementary data for a particular unit given the unit id
-    and the dataset id."""
-    data = database.get_data(dataset_id=dataset_id, unit_id=unit_id)
+    return OK
+
+
+@app.get(DATASETS_PATH)
+async def get_datasets(survey_id: str):
+    data = database.get_datasets(survey_id)
     return data
 
 
-@app.post("/dataset_schema")
-async def publish_schema(
-    dataset_schema_id: str, survey_id: str, payload: dict = Body(...)
-):
-    """Put a schema into the database and return the schema_id."""
-    version = database.set_schema(dataset_schema_id, survey_id, payload)
-    return {"dataset_schema_id": dataset_schema_id, "version": version}
+@app.get(UNIT_DATA_PATH)
+async def get_unit_data(dataset_id: str, unit_id: str):
+    data = database.get_data(dataset_id=dataset_id, unit_id=unit_id)
```

Comment:
Hi James, could you please add doc-stings to any methods that don't have any. the strand will be as follows. (I haven't had a chance to chat about our ways of working, but will book something in.

"""Summary or Description of the Function
 
 Parameters:
 argument1 (int): Description of arg1
 
 Returns:
 int:Returning value
 
"""